pro_tes/middleware/models.py

```diff
@@ -1,31 +1,35 @@
 """Model for the Access Uri Combination."""
-# pylint: disable=no-name-in-module
+
+# pragma pylint: disable=no-name-in-module
+
 from typing import List
-from pydantic import BaseModel
 
+from pydantic import AnyUrl, BaseModel, HttpUrl
 
 # pragma pylint: disable=too-few-public-methods
```

Comment:
As I said before: don't use module-level pragmas unless you have a good reason, e.g., disabling `too-few-public-methods` totally makes sense in a module full of models. But it's not clear to me why `no-name-in-module` should be disabled in every line of the module. Unless I'm missing something, please disable the linter rule in line, wherever you need it.

If I _am_ missing something, and you really need to disable this in more than two places (and possibly more in the future), then please merge it with the other pragma further below.

Module organization should always be:
- 1st: Module-level docstring
- 2nd: Imports
- Everything else

Reply:
Actually, the error was arising from one of the import lines which is `from pydantic import AnyUrl, BaseModel, HttpUrl` that's why I have used `no-name-in-module` before imports. Maybe I should disable this in the same line as you mentioned above.